.claude/skills/issue.md

```
@@ -0,0 +1,178 @@
+<br><br>
+
+## 참고자료
+
+{관련 스크린샷, 에러 로그 등 — 없으면 생략}
+```
+
+---
+
+## 브랜치 네이밍 컨벤션
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)